default.nix

```diff
@@ -70,8 +70,6 @@ rec {
     inherit (plutus-apps) fixStylishHaskell fix-purs-tidy fixPngOptimization fixCabalFmt;
     inherit plutus-playground web-ghc;
     src = ./.;
-    play-generated = plutus-playground.generated-purescript;
-    nami-generated = pab-nami-demo.generated-purescript;
   };
```

Comment:
We no longer check the generated Purs in CI. (Again this was not being done anyway)